addons/point_of_sale/static/tests/tours/helpers/utils.js

```diff
@@ -25,3 +26,42 @@ export function negateStep(step) {
         trigger: negate(step.trigger),
     };
 }
+
+export function refresh() {
+    return {
+        content: `refresh page`,
+        trigger: "body",
+        run: () => {
+            window.location.reload();
+        },
+    };
+}
+
+/**
+ * Verifies the non-existence of an element specified by a CSS selector on the page. If a text value is provided,
+ * the function also checks that no elements matching the selector contain the specified text.
+ *
+ * @param {string} selector - The CSS selector of the element to be verified. Must be a valid selector that jQuery can process.
+ * @param {string} [value] - An optional text value that the element found by the selector is supposed to contain. This check is only performed if the value is provided.
+ * @example
+ * // To verify the non-existence of an element with the class 'myClass':
+ * verifyElementDoesNotExist('.myClass');
+ *
+ * @example
+ * // To verify the non-existence of an element with the class 'button-floor' and containing the text 'MyFloor':
+ * verifyElementDoesNotExist('.floor-selector .button-floor', 'MyFloor');
+ */
+export function elementDoesNotExist(selector, value) {
+    return {
+        content: `Check that element "${selector} ${value}" don't exist.`,
+        trigger: "body",
```

Comment:
here the idiomatic way to achieve this would be to put your constructed selector as the `trigger`, something like
```js
trigger: negate(selector),
isCheck: true,
```
this way you don't need to write a function

Reply:
I modify the elementDoesNotExist with your trigger. I believe that's could be a usefull function because I suppose we will have to check often if something don't exist and in that way, that could be done in only one line. what do you think about ?